libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -133,7 +144,7 @@ void PbftSyncPacketHandler::process(const PacketData &packet_data, const std::sh
   LOG(log_tr_) << "Synced PBFT block hash " << pbft_blk_hash << " with " << sync_block.cert_votes.size()
                << " cert votes";
   LOG(log_tr_) << "Synced PBFT block " << sync_block;
-  pbft_mgr_->syncBlockQueuePush(std::move(sync_block), packet_data.from_node_id_);
+  pbft_mgr_->syncBlockQueuePush(std::move(sync_block), packet_data.from_node_id_, std::move(cert_votes));
```

Comment:
@JakubFornadel ask that what happens if cert_votes is empty due to pbft_chain_synced == false ?

Reply:
I already know what happens, so you dont need to ask it in my name :) Either you are interested in answer or delete this